src/Maestro/SubscriptionActorService/PullRequestDescriptionBuilder.cs

```diff
@@ -40,110 +41,149 @@ public PullRequestDescriptionBuilder(ILoggerFactory loggerFactory, string descri
     /// <param name="deps">Dependencies updated</param>
     /// <param name="committedFiles">List of commited files</param>
     /// <param name="build">Build</param>
-    /// <returns>Task</returns>
     /// <remarks>
     ///     Because PRs tend to be live for short periods of time, we can put more information
     ///     in the description than the commit message without worrying that links will go stale.
     /// </remarks>
-    public void AppendBuildDescription(UpdateAssetsParameters update, List<DependencyUpdate> deps, List<GitFile> committedFiles, Build build)
+    public void AppendBuildDescription(UpdateAssetsParameters update, List<DependencyUpdate> deps, List<GitFile>? committedFiles, Build build)
     {
         var changesLinks = new List<string>();
 
-        //Find the Coherency section of the PR description
-        if (update.IsCoherencyUpdate)
+        string sourceRepository = update.SourceRepo;
+        Guid updateSubscriptionId = update.SubscriptionId;
+        string sectionStartMarker = $"[marker]: <> (Begin:{updateSubscriptionId})";
+        string sectionEndMarker = $"[marker]: <> (End:{updateSubscriptionId})";
+        int sectionStartIndex = RemovePRDescriptionSection(sectionStartMarker, sectionEndMarker);
+
+        var subscriptionSection = new StringBuilder();
+        subscriptionSection.AppendLine(sectionStartMarker);
+        subscriptionSection.AppendLine($"## From {sourceRepository}");
+        subscriptionSection.AppendLine($"- **Subscription**: {updateSubscriptionId}");
+        subscriptionSection.AppendLine($"- **Build**: {build.AzureDevOpsBuildNumber}");
+        subscriptionSection.AppendLine($"- **Date Produced**: {build.DateProduced.ToUniversalTime():MMMM d, yyyy h:mm:ss tt UTC}");
+        // This is duplicated from the files changed, but is easier to read here.
+        subscriptionSection.AppendLine($"- **Commit**: {build.Commit}");
```

Comment:
```suggestion
        subscriptionSection.AppendLine(sectionStartMarker)
            .AppendLine($"## From {sourceRepository}")
            .AppendLine($"- **Subscription**: {updateSubscriptionId}")
            .AppendLine($"- **Build**: {build.AzureDevOpsBuildNumber}")
            .AppendLine($"- **Date Produced**: {build.DateProduced.ToUniversalTime():MMMM d, yyyy h:mm:ss tt UTC}");
        // This is duplicated from the files changed, but is easier to read here.
        subscriptionSection.AppendLine($"- **Commit**: {build.Commit}");
```

Reply:
I'm not sure if I did this correctly, I couldn't find a way to do `tab` so I did 4 spaces instead